### PR TITLE
Updating katello test configuration

### PR DESCRIPTION
**Pull request by daviddavis**
Cleaning up some of the code.

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_katello.sh [hunk 1]
```diff
@@ -40,13 +40,7 @@ gem install bundler --no-ri --no-rdoc
...
 
 # Retry as rubygems (being external to us) can be intermittent
-while ! bundle install --without development -j 5; do
-  (( c += 1 ))
-  if [ $c -ge 5 ]; then
-    echo "bundle install continually failed" >&2
-    exit 1
-  fi
-done
+bundle install --without development -j 5 --retry 5
```
Comment (daviddavis) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_katello.sh]: Small refactoring.

File: puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_plugin.sh [hunk 2]
```diff
@@ -47,9 +47,6 @@ bundle exec rake db:drop db:create db:migrate
 # Now let's introduce the plugin
 echo "gem '${plugin_name}', :path => '${PLUGIN_ROOT}'" >> bundler.d/Gemfile.local.rb
 
-# Plugin specifics..
-[ -e ${PLUGIN_ROOT}/script/ci/katello.yml ] && cp ${PLUGIN_ROOT}/script/ci/katello.yml ${PLUGIN_ROOT}/config/katello.yml
```
Comment (daviddavis) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_plugin.sh]: AFAIK, we don't need this for katello. I'm actually trying to remove `katello.yml` from our repo which prompted this PR.

Comment (daviddavis) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_plugin.sh]: AFAIK, we don't need this for katello. I'm actually trying to remove script/ci/katello.yml from our repo which prompted this PR.

**Comment by domcleal** [comment]
Thanks @daviddavis.

**Pull request merged**